### PR TITLE
[DO-2055] Added method for find third person

### PR DESCRIPTION
**Pull request by EmmaSamoilova**

### CODE REVIEW

File: business_register/converter/declaration.py [hunk 1]
```diff
@@ -94,6 +94,71 @@ def find_person(self, pep_id):
             self.log_error(f'Cannot find person with nacp_id {pep_id}')
         return None
 
+    def find_third_person_rights(self, right_data, name_of_rights):
+        OTHER_PERSON_TYPES = {
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py:98]: Consider initializing that in the __init__() method of the DeclarationConverter class. Then we can use it in all other classes

File: business_register/converter/declaration.py [hunk 2]
```diff
@@ -94,6 +94,71 @@ def find_person(self, pep_id):
...
+        elif owner_type == BaseRight.FOREIGN_CITIZEN:
+            ukr_full_name = right_data.get('ukr_fullname')
+            if ukr_full_name in self.NO_DATA:
+                ukr_full_name = ''
+            eng_full_name = right_data.get('eng_fullname')
+            if eng_full_name in self.NO_DATA:
+                eng_full_name = ''
+            full_name = ukr_full_name if ukr_full_name else eng_full_name
+        elif owner_type == BaseRight.UKRAINE_LEGAL_ENTITY:
+            full_name = right_data.get('ua_company_name')
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py:126]: Only a person can have the full name). Maybe, company name?

**Review by IegorSoboliev — CHANGES_REQUESTED** [review]

File: business_register/converter/declaration.py (hunk 1 above)
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py]: done

File: business_register/converter/declaration.py (hunk 2 above)
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py]: done

**Review by IegorSoboliev — APPROVED** [review]

**Review by OlexandrTopuzov — APPROVED** [review]

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**